src/components/NavBase.vue

```diff
@@ -226,6 +232,7 @@ export default {
       this.isTransitioning = false;
       if (this.isOpen) {
         this.$emit('opened');
+        if (this.preventFocusTrap) return;
```

Comment:
should we also prevent scroll locking? At this point we are going into mobile viewport territory.